src/Components/Attributes/Attributes.js

```diff
@@ -105,7 +109,17 @@ class Attributes extends React.Component {
     // `dataStructures` prop is optional and is used to
     // resolve inheritance, references and includes/mixins, plus to
     // render the inheritance tree.
-    const dataStructures = props.dataStructures || [];
+    let dataStructures = props.dataStructures || [];
+    let element = props.element;
+
+    if (isFunction(element.getMetaProperty)) {
+      // serialize minim Element inputs (`element` and `dataStructures`)
+      // to the expected refract 0.6 serialization
+      const refractSerializer = new JSON06Serialiser(minim.namespace());
+      element = refractSerializer.serialise(element).content[0];
+      dataStructures = dataStructures.map(d => refractSerializer.serialise(d).content[0]);
```

Comment:
Should this be wrapped in an `if (dataStructures instanceof minim.Element)` check too? I wonder if user provides inconsistencies in if the element is an element object vs serialised JSON.

```js
const refractSerializer = new JSON06Serialiser(minim.namespace());

if (element instanceof minim.Element) {
  element = refractSerializer.serialise(element).content[0];
}

if (dataStructures instanceof minim.Element) {
  dataStructures = dataStructures.map(d => refractSerializer.serialise(d).content[0]);
}
```